src/main/java/Airport.java

```diff
@@ -0,0 +1,137 @@
+import Planes.ExperimentalPlane;
+import Planes.MilitaryPlane;
+import Planes.PassengerPlane;
+import Planes.Plane;
+import exceptions.NonMilitaryPlainException;
+import models.MilitaryType;
+
+import java.util.*;
+
+
+public class Airport {
+    private List<? extends Plane> planes;
+
+
+    public List<PassengerPlane> getPassengersPlane() {
+        List<? extends Plane> planesList = this.planes;
+        List<PassengerPlane> passengersPlaneList = new ArrayList<>();
+        for (Plane plane : planesList) {
+            if (plane instanceof PassengerPlane) {
+                passengersPlaneList.add((PassengerPlane) plane);
+            }
+        }
+        return passengersPlaneList;
+    }
+
+    public List<MilitaryPlane> getMilitaryPlanes() throws NonMilitaryPlainException {
+        List<MilitaryPlane> militaryPlanes = new ArrayList<MilitaryPlane>();
+        for (Plane plane : planes) {
+            if (plane instanceof MilitaryPlane) {
+                militaryPlanes.add((MilitaryPlane) plane);
+            } else {
+                throw new NonMilitaryPlainException("Plane is not military");
```

Comment:
exceptions aren't handled in the app. if it just thrown here, the app will crash just if there is a non-military plain is found in iterated collection.